BMM.Core/Implementations/PlayObserver/PlayStatistics.cs

```diff
@@ -241,11 +243,11 @@ public void Clear()
             PortionsListened.Clear();
         }
 
-        public TrackPlayedEvent ComposeEvent(PlayMeasurements measurements)
+        public TrackPlayedEvent ComposeEvent(PlayMeasurements measurements, [CallerMemberName] string callerName = "")
         {
             var user = _userStorage.GetUser();
             
-            return new TrackPlayedEvent
+            var tpe = new TrackPlayedEvent
```

Comment:
I definitely stumbled over "tpe". maybe just "e" is a better name.

Reply:
In PlaystatisticsDecorator "ev" is used. which I think is also better.

Reply:
Changed to 'ev'